pantalaimon/daemon.py

```diff
@@ -851,9 +853,47 @@ async def send_message(self, request):
 
         # The room isn't encrypted just forward the message.
         if not encrypt:
+            content_msgtype = ""
```

Comment:
This method is already quite big, splitting out this functionality into a separate method will make this a bit more readable.